lib/ex_twilio/resources/insights/settings.ex

```diff
@@ -1,4 +1,4 @@
-defmodule ExTwilio.Insights.Voice.Settings do
+defmodule ExTwilio.Insights.Settings do
```

Comment:
If we were doing a versioned library I'd be wary of changing an existing module like this, but I think this is fine for now